modules/ROOT/pages/export-app-data-to-csv.adoc

```diff
@@ -0,0 +1,34 @@
+= Export App Data to a CSV File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In built-in dashboards, you can export app data to a CSV file, which you can then use in other applications. The data export works on an individual chart level, so only data in the specified chart is exported. When you apply a filter to a dashboard, the exported data includes only the data included in the filter you set. For example, if you apply a time period filter, only the data for that time period is exported.   
+
+[NOTE]
+This feature requires a Titanium subscription.
+
+. In Anypoint Monitoring, click *Built-in dashboards* in the sidebar on the left. 
+. Select the application for which to download the data. 
+. Specify the time range for the data to download. 
+. In the chart that contains the data to download, select *Export CSV*. 
```

Comment:
Shouldn't we clarify that before that the user has to click on the 3-dot button?

Reply:
Okay!